### PR TITLE
[ALLUXIO-2793]  Use precondition check chaining in BlockInfo

### PR DESCRIPTION
**Pull request by yuluo-ding**
[ALLUXIO-2793](https://alluxio.atlassian.net/browse/ALLUXIO-2793)

`mLocations = new ArrayList<>(Preconditions.checkNotNull(locations));`

### CODE REVIEW

**Comment by yuluo-ding** [comment]
alluxio-bot, check this please

**Comment by yuluo-ding** [comment]
alluxio-bot, check this please

**Comment by yuluo-ding** [comment]
alluxio-bot, check this please

File: core/common/src/main/java/alluxio/wire/BlockInfo.java [hunk 1]
```diff
@@ -97,8 +97,9 @@ public BlockInfo setLength(long length) {
    * @return the block information
    */
   public BlockInfo setLocations(List<BlockLocation> locations) {
-    Preconditions.checkNotNull(locations);
-    mLocations = new ArrayList<>(locations);
+    // Preconditions.checkNotNull(locations);
```
Comment (ifcharming) [inline core/common/src/main/java/alluxio/wire/BlockInfo.java]: Can you please remove those two lines instead of commenting them out?

**Comment by yuluo-ding** [comment]
alluxio-bot, check this please

File: core/common/src/main/java/alluxio/wire/BlockInfo.java (hunk 1 above)
Comment (yuluo-ding) [inline core/common/src/main/java/alluxio/wire/BlockInfo.java]: I have modified it.

**Comment by ifcharming** [comment]
Jenkins, test this please

**Pull request merged**